app/controlplane/api/workflowcontract/v1/crafting_schema.proto

```diff
@@ -149,18 +150,14 @@ message PolicyAttachment {
   // set to true to disable this rule
   bool disabled = 4;
 
-  repeated PolicyArgument with = 5;
+  // optional arguments for policies
+  map<string, string> with = 5;
+//  bytes with = 5;
```

Comment:
spurious comment

I'd also add a comment here showing some examples of how to use it, containing single, multiline or comma separated? what do you think?